main.go

```diff
@@ -127,9 +127,10 @@ func main() {
 		err = cmd.Execute(command, title, assumeYes, maxWait, filename)
 
 	case "play":
+		maxWait := uintArg(args, "--max-wait", cfg.RecordMaxWait())
```

Comment:
More than happy to make a new cfg. option for this, and I agree making it default to a reasonable value (say, 2s) makes sense.